CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Fixed 🐞
 
 -   Update slider of area metric options correctly on changes of related input field [#2787](https://github.com/MaibornWolff/codecharta/pull/2787)
+-   Limit length of labels in legend panel #2329
```

Comment:
This GitHub linking doesn't work with the changes information we display to the user within CodeCharta, which are generated by this changelog. Although we might add this parsing to the generation script.

We link the PR and not the issue. Feel very free to address that. I find it more natural to link the issue as well :)

```suggestion
-   Limit length of labels in legend panel [#2804](https://github.com/MaibornWolff/codecharta/pull/2804)
```